docs/pages/x.tsx

```diff
@@ -3,22 +3,23 @@ import Divider from '@mui/material/Divider';
 import Head from 'docs/src/modules/components/Head';
```

Comment:
Offtopic: Is this intentional? 🙂

<img width="326" src="https://github.com/mui/material-ui/assets/13808724/6fecd1c5-37d8-4611-ab36-d048e507ba1c">